### DISCUSSION TITLE
fortnite api

### DISCUSSION THREAD

**Discussion by gromaxbro** [LeleDerGrasshalmi/FortniteEndpointsDocumentation#256] [General]
url = "https://fngw-mcp-gc-livefn.ol.epicgames.com/fortnite/api/game/v2/profile/84919aee13c0483c8d614d1e1d92fe78/client/Individual"

# Define the query parameters
params = {
    "profileId": "athena",  # Battle Royale data
    "rvn": -1
}

# Define the headers
headers = {
    "Content-Type": "application/json",
    "Authorization": "bearer "+res.json()["access_token"],
}

# Make the POST request
response = requests.post(url, headers=headers, params=params)
print(response.text)

error:

  "errorCode" : "errors.com.epicgames.fortnite.operation_not_found",
  "errorMessage" : "Operation Individual not valid",
  "messageVars" : [ "Individual" ],
  "numericErrorCode" : 16035,
  "originatingService" : "fortnite",
  "intent" : "prod-live"

**Comment by LeleDerGrasshalmi** [comment]
Error looks pretty clear to me 🤔

**Reply by uwUSKibidi** [reply]
Haha
